darts/models/forecasting/tide_model.py

```diff
@@ -424,11 +434,15 @@ def __init__(
         hidden_size
             The width of the layers in the residual blocks of the encoder and decoder.
         temporal_width_past
-            The width of the layers in the past covariate projection residual block. If `0`,
+            The width of the output layer in the past covariate projection residual block. If `0`,
             will bypass feature projection and use the raw feature data.
         temporal_width_future
-            The width of the layers in the future covariate projection residual block. If `0`,
+            The width of the output layer in the future covariate projection residual block. If `0`,
             will bypass feature projection and use the raw feature data.
+        temporal_hidden_size_past
```

Comment:
mention here that by default, it uses the same value as `temporal_width_*`, but that it is recommended setting a lower value.

Reply:
Change made